packages/cli/src/credentials/credentials.controller.ee.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-cycle */
```

Comment:
Review for cred services and controllers here: https://github.com/n8n-io/n8n/pull/3601